processes/developmentSetup.py

```diff
@@ -70,16 +71,24 @@ def initializeDB(self):
         with self.adminDBConnection.engine.connect() as conn:
             conn.connection.set_isolation_level(ISOLATION_LEVEL_AUTOCOMMIT)
             try:
-                conn.execute('CREATE DATABASE {}'.format(os.environ['POSTGRES_NAME']))
+                conn.execute(
+                    sa.text(f"CREATE DATABASE {os.environ['POSTGRES_NAME']}"),
+                )
             except ProgrammingError:
                 pass
 
             try:
-                conn.execute('CREATE USER {} WITH PASSWORD \'{}\''.format(
-                    os.environ['POSTGRES_USER'], os.environ['POSTGRES_PSWD']
-                ))
-                conn.execute('GRANT ALL PRIVILEGES ON DATABASE {} TO {}'.format(
-                    os.environ['POSTGRES_NAME'], os.environ['POSTGRES_USER'])
+                conn.execute(
+                    sa.text(
+                        f"CREATE USER {os.environ['POSTGRES_USER']} "
+                        f"WITH PASSWORD '{os.environ['POSTGRES_PSWD']}'",
+                    ),
+                )
+                conn.execute(
+                    sa.text(
+                        f"GRANT ALL PRIVILEGES ON DATABASE {os.environ['POSTGRES_NAME']} "
+                        f"TO {os.environ['POSTGRES_USER']}",
+                    ),
```

Comment:
So when you use two f strings in the sa.text field, they will combine into one string? I'm assuming you did it like this since you don't want one very long f string.

Reply:
pretty much, though I'm really just passing one string to `text` (notice the lack of comma between the two lines). Python automatically interprets this as one string, and yep, did it so that it's not one huge line!